packages/ui/Button.js

```diff
@@ -1,5 +1,8 @@
+import MuiButton from "@mui/material/Button";
```

Comment:
This cancels out my work 👀 A heads up would have been appreciated lol @kilemensi

Reply:
... This is why I haven't merged this PR @gertie-sheshe. Waiting for you to merge yours.

Reply:
PR merged